src/domain/chainContext.ts

```diff
@@ -13,6 +16,8 @@ import { composableCowContract, DBService, getLogger } from "../utils";
 const WATCHDOG_FREQUENCY = 5 * 1000; // 5 seconds
 const WATCHDOG_KILL_THRESHOLD = 30 * 1000; // 30 seconds
 
+const MULTICALL3 = "0xcA11bde05977b3631167028862bE2a173976CA11";
```

Comment:
I think this logic should be moved eventually to the SDK

Reply:
Eventually, maybe. But we need a place to point the Valory guys to for a commit hash, which we have now, for what Valory should be doing :slightly_smiling_face: